ddtrace/contrib/celery/util.py

```diff
@@ -0,0 +1,31 @@
+# Project
+from ddtrace import Pin
+
+# Service info
+APP = 'celery'
+SERVICE = 'celery'
+
+
+def meta_from_context(context):
+    """ helper to extract meta values from a celery context """
+    meta_keys = (
+        'called_directly', 'correlation_id', 'delivery_info', 'eta', 'expires', 'hostname',
```

Comment:
Makes sense.

We can reasonably drop `called_directly`, and `is_eager`. Those should be clear from the trace hierarchy anyways (if they show up under a flask request, they were called directly or were eager... compared to their own trace, which is run by a worker).

We can probably drop the `task` name, which is the name used for the resource anyways.

We can drop `eta`, `expires`, `timelimit` when they are `None` (or `(None, None)`).

Lastly, we can drop `utc` since it is configured globally.


This will leave us with:

* `correlation_id`
* `delivery_info`
* `eta` - When not `None`
* `expires` - When not `None`
* `hostname`
* `id`
* `reply_to`
* `retries` - Maybe even drop this when it is `0`
* `timelimit` - When not `(None, None)`


This seem more reasonable? Anything else in here that seems like a likely candidate to remove?